stock_reserve_rule/models/stock_reserve_rule.py

```diff
@@ -152,6 +154,92 @@ class StockReserveRuleRemoval(models.Model):
         "When empty, any packaging can be removed.",
     )
 
+    TOLERANCE_LIMIT = [
+        ("no_tolerance", "No tolerance"),
+        ("upper_limit", "Upper Limit"),
+        ("lower_limit", "Lower Limit"),
+    ]
+
+    tolerance_requested_limit = fields.Selection(
```

Comment:
@geomer198 Maybe some help entries for those fields ?

Reply:
Resolved

Reply:
@rousseldenis good now?